hw2.sql

```diff
@@ -0,0 +1,161 @@
+--Formatted with https://www.dpriver.com/pp/sqlformat.htm
```

Comment:
The script was failing when the interpreter tried reading your comments.  I think [this documentation](https://mariadb.com/kb/en/comment-syntax/) explains why.  Apparently in MySQL/MariaDB, you need to add a space after dashes with this comment style '-- '.

Reply:
Forgot to mention, I am using MySQL. Added note in new commit